layouts/partials/navigation.html

```diff
@@ -5,26 +5,21 @@
       {{ $logoWhite:= site.Params.logo_white }}
       {{ $orgName:= site.Params.brand.parent_org_name }}
       {{ $orgUrl:= site.Params.brand.parent_org_url }}
-      <a class="navbar-brand px-2"
-        href="{{ if (or $logo $logoWhite) }}{{ $orgUrl }}{{else}}{{ site.BaseURL | relLangURL }}{{ end }}">
-        {{ if (or $logo $logoWhite) }}
-        {{ if .IsHome }}
+     
+       
         <div class="text-center">
-          <img class="img-fluid d-inline"
+          <a class="navbar-brand "
+          href="{{ if (or $logo $logoWhite) }}{{ $orgUrl }}{{else}}{{ site.BaseURL | relLangURL }}{{ end }}">
+          {{ if (or $logo $logoWhite) }} 
+          <img class="img-fluid d-inline globalLogoImg"
             src="{{if $logoWhite }}{{ $logoWhite | absURL }} {{ else }} {{ $logo | absURL }}{{ end }}"
-            alt="{{ site.Title }}"> <a class="text-white d-block" href="{{ site.BaseURL | relLangURL }}">{{ site.Title
+            alt="{{ site.Title }}">
+            </a>
+             <a class="text-white d-block" href="{{ site.BaseURL | relLangURL }}">{{ site.Title
```

Comment:
Something isn't right here. If a site has no logo in place, this changes the top navigation. Please take a look again and test this change with and without a logo set:

![Screenshot of the UNICEF Inventory theme site without a logo set. The top navigation bar is shifted and does not look right.](https://user-images.githubusercontent.com/4721034/180355535-e7061249-067e-4de8-8e3e-2ca506768c40.png "Screenshot of the UNICEF Inventory theme site without a logo set. The top navigation bar is shifted and does not look right.")

Reply:
Resolved in commit 1f42c23b9bdf850505370e28efae7d3d15c899b1.